### DISCUSSION TITLE
Want to import forms style for just the checkbox and not the other input elements

### DISCUSSION THREAD

**Discussion by abhishekvash-tl** [tailwindlabs/tailwindcss#9862] [Help]
I dont want the default form plugin styles for any element but the checkbox. Is preferential import possible ?

**Comment by wongjn** [comment]
If you are talking about the `@tailwindcss/forms` plugin, you could use the [class-based styling strategy](https://github.com/tailwindlabs/tailwindcss-forms#using-classes-to-style) to only style checkboxes, by adding the `form-checkbox` class on them. Otherwise, if you are talking about styles from the core preflight plugin, you could [disable the plugin](https://tailwindcss.com/docs/preflight#disabling-preflight) and re-add the styles you'd like but scoped to checkbox inputs.
